tcms/xmlrpc/api/testcase.py

```diff
@@ -212,8 +212,8 @@ def add_tag(case_id, tag):
         :raises: PermissionDenied if missing *testcases.add_testcasetag* permission
         :raises: TestCase.DoesNotExist if object specified by PK doesn't exist
     """
-    t, _ = Tag.objects.get_or_create(name=tag)
-    TestCase.objects.get(pk=case_id).add_tag(t)
+    tag, _ = Tag.objects.get_or_create(name=tag)
```

Comment:
Doesn't pylint complain that you are overriding the `tag` parameter ?

Reply:
No, it doesn't. I suppose it thinks that I do not shadow it, but rather give another value to the parameter that is being passed.